src/main/java/gov/cms/madie/cql_elm_translator/dto/CqlLookups.java

```diff
@@ -0,0 +1,30 @@
+package gov.cms.madie.cql_elm_translator.dto;
+
+import gov.cms.madie.cql_elm_translator.utils.cql.parsing.model.CQLCode;
+import gov.cms.madie.cql_elm_translator.utils.cql.parsing.model.CQLCodeSystem;
+import gov.cms.madie.cql_elm_translator.utils.cql.parsing.model.CQLDefinition;
+import gov.cms.madie.cql_elm_translator.utils.cql.parsing.model.CQLIncludeLibrary;
+import gov.cms.madie.cql_elm_translator.utils.cql.parsing.model.CQLParameter;
+import gov.cms.madie.cql_elm_translator.utils.cql.parsing.model.CQLValueSet;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.Set;
+
+@Data
+@Builder
+public class CqlLookups {
```

Comment:
I think this is QDM specific class. Can live in QDM translator

Reply:
@adongare Moved it out and back to madie-qdm-elm-translation.